javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -62,13 +62,15 @@ public Javers javers() {
     }
 
     @Bean(name = "authorProvider")
+    @ConditionalOnMissingBean
```

Comment:
Hmm, to chyba powinniśmy zmienić na compile, bo z co mi tłumaczył Kuba to ten warunek springowy na obecność klasy na classpath działa niedererministycznie....